### DISCUSSION TITLE
Update.exe dosent work [REQ]

### DISCUSSION THREAD

**Discussion by Xaos-biker** [Chia-Network/chia-blockchain#3270] [Support]
How i can update in Windows my gui trough update.exe. I try different command, but nothing heppening.

**Comment by anxelbao** [comment]
I try the same thing and get nothing. How is it possible that we are almost in the middle of 2022 and this application does not have a self-boosting system as the most ridiculous applications that exist have? I am about to stop using CHIA-NETWORK because it is deficient in almost every way. I don't know what the developers are thinking about.

**Reply by loppefaaret** [reply]
do you mean auto update function ? it is an attack vector, and would also possibly be seen as a way for Chia Network to control proposed updates / forks to the Chia blockchain...

**Reply by anxelbao** [reply]
ok, then simply a warning in the interface so that the user can update with a click. I don't see the difference in having to do it manually. The current problem is that many updates leave the installed application offline and you need to update so that chia continues to do its job. This forces you to check the application every few hours in case you have decided to release new software and thus update it manually... it seems to me something, at the very least, Archaic.

**Comment by loppefaaret** [comment]
> How i can update in Windows my gui trough update.exe. I try different command, but nothing heppening.

you should simply close down the Chia client, and run the new version ofthe installer, and it should remove the current install, and install the new version

can you possibly describe exactly what part of this process is failing ?
